codes/quantum/qudits/qudit_stabilizer.yml

```diff
@@ -8,20 +8,20 @@ physical: qudits
 logical: qudits
 
 name: '\(\mathbb{Z}_q\)-qudit stabilizer code'
-#introduced: '\cite{arXiv:quant-ph/9705052}'
+introduced: '\cite{arXiv:quant-ph/9705052}'
 
-description: 'An \(((n,p^k,d))_q\) (with \(q\) not necessarily prime) stabilizer code is denoted as \([[n,k]]_q\) or \([[n,k,d]]_q\), where \(d\) is the code''s distance. Stub.'
-#Logical subspace is the joint \(+1\) eigenspace of a set of \(p^{n-k}\) commuting qudit Pauli operators forming the code''s stabilizer group, which cannot contain \(-I\).
-#The distance is the minimum weight of an error that implements a nontrivial logical operation on the code.
-#Normalizer \(N(S)\) of the stabilizer \(S\) is the set that contains all operators that commutes with \(S\).
-#The stabilizer code can correct errors \(\varepsilon \in P_n(p)\) if and only if \(\hat{E}^\dagger \hat{F} \notin \hat{N}(S)\setminus \hat{S}\) for all \(E,F \in \varepsilon\).'
-# Defined only for prime-power qudits for now
-# Galois qudit codes have lifted stabiilizers, generally k=n-r/m
+description: 'An \(((n,q^k,d))_q\) (with \(q\gt 1\)) stabilizer code is denoted as \([[n,k]]_q\) or \([[n,k,d]]_q\), where \(d\) is the code''s distance.
```

Comment:
Thanks for the paper you found. It provides a "size" theorem relating log dim K to order of stab group |S|, K = q^n / |S|. Since need not be q^r, K need not be q^{n-r} for composite q. Please change this to account for that.

Reply:
Done.